.github/scripts/prepare_release.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Fail on first error.
+set -e
+
+releaseLevel="$1"
+
+oldVersion="$(node -pe 'require("./package.json").version')"
+npx standard-version@"$STD_VERSION_VERSION" --release-as "$releaseLevel" --skip.commit=true --skip.changelog=true --skip.tag=true
+newVersion="$(node -pe 'require("./package.json").version')"
+
+sed -i -e "s/  VERSION\\s*=\\s*\"$oldVersion\"/  VERSION = \"$newVersion\"/" version.rb
+
+npx conventional-changelog-cli@"$CHANGELOG_VERSION" -p angular -i CHANGELOG.md -s
```

Comment:
What's the reason for skipping changelog generation  in `standard-version` but then doing it here?

Reply:
There was a point where the version of standart-version and conventional-changelog-cli was set via environmental variable. In that case it was necessary to separate the steps. This has been changed.